.github/workflows/wheel_manylinux_pypi_nightly.yaml

```
@@ -0,0 +1,52 @@
+# GH actions.
+name: Wheel-Manylinux-Pypi
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

```suggestion
  workflow_dispatch:
  pull_request:
```

it's also nice just to have a button to press whenever we need to

---

I didn't know that! Will make this change, thanks.